Project.toml

```
@@ -12,7 +12,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-Makie = "0.20"
+Makie = ">= 0.20"
```

```suggestion
Makie = "0.20, 0.21"
```
since every `0.x` release is potentially breaking, according to semver, we really need to list them separately here.